src/engine/SCons/Variables/PathVariable.py

```diff
@@ -78,11 +78,13 @@
 
 class _PathVariableClass(object):
 
-    def PathAccept(self, key, val, env):
+    @staticmethod
+    def PathAccept(key, val, env):
```

Comment:
Doesn't this change how it's allowed to be called?

Reply:
not at an API level, just internally to Python, which no longer passes an instance to it - which it doesn't use, so staticmethod is appropriate - and no longer needs to bind it to each instance.